1_check_dependencies.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eo pipefail
+
+. utils.sh
+
+check_env_var "UNIQUE_TEST_ID"
+
+check_env_var "CONJUR_APPLIANCE_IMAGE"
+check_env_var "CONJUR_FOLLOWER_COUNT"
+check_env_var "CONJUR_ACCOUNT"
+check_env_var "AUTHENTICATOR_ID"
+check_env_var "CONJUR_ADMIN_PASSWORD"
+check_env_var "CONJUR_NAMESPACE_NAME"
+
+check_env_var "APP_NAMESPACE_NAME"
+
+if [[ "$PLATFORM" == "openshift" && "$DEV" == "false" ]]; then
+  check_env_var "DOCKER_REGISTRY_PATH"
+  check_env_var "OPENSHIFT_USERNAME"
+  check_env_var "OPENSHIFT_PASSWORD"
+elif [[ "$PLATFORM" = "kubernetes" ]]; then
+    if [ "${DEV}" = "false" ]; then
```

Comment:
Why you are not using == like line 17?
both IF relevant only if DEV=false so you can do :
IF [ "${DEV}" == "false" ]; then
   if [[ "$PLATFORM" == "openshift"] ;then
     .......
   elif [[ "$PLATFORM" = "kubernetes" ]]; then
    ....
  fi

Reply:
That is a good question.. I will update